src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/solicitorcreatecase/SolicitorCreateContestedAboutToSubmitHandler.java

```diff
@@ -82,6 +83,15 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
 
         representationWorkflowService.persistDefaultOrganisationPolicy(caseData);
 
+        // updates the tab version of applicantInRefuge with the question version, then removes question version.
+        Optional.ofNullable(caseData.getRefugeWrapper().getApplicantInRefugeQuestion())
+                .ifPresent(question -> {
+                    caseData.getRefugeWrapper().setApplicantInRefugeTab(question);
+                    log.info("Updating applicantInRefugeTab for case reference {}. Removing applicantInRefugeQuestion",
```

Comment:
I question whether this log message is of any value to us. Logs do actually cost money.

Reply:
@jthmcts Thanks for raising.  Will contact you to discuss.